app/views/report/_form_columns.html.haml

```diff
@@ -26,6 +26,13 @@
       = render :partial => "column_lists"
     %strong
       = _('* Caution: Changing these fields will clear all selected values below them!')
+    - if @edit[:new][:model] && @edit[:new][:model].starts_with?('Chargeback')
+      - unavailable_fields = unavailable_fields_for_model
+      - if unavailable_fields
+        - report_base = @edit[:new][:model].titleize.split(' ').last.pluralize
+        %br
+        %strong
+          = _("* Caution: #{unavailable_fields} are not supported for Chargeback for #{report_base}.")
```

Comment:
This is probably not okay from the translation side, you need to use the `%` operator.